.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+To report a good issue, please verify that you validate these points:
+
+- [ ] you read the [documentation](../../#available-commands--options) and the [some advanced use cases](../../#more-advanced-use-cases)
```

Comment:
maybe `you have read the documentation about [command line options](../../#available-commands--options) and [advanced use cases](../../#more-advanced-use-cases)`?